beetmoverscript/docker.d/worker.yml

```diff
@@ -131,6 +145,14 @@ bucket_config:
           buckets:
             nightly_components: 'maven-nightly-stage-s3-upload-bucket'
         dummy: {}
+        dep:
+          credentials:
+            id: { "$eval": "DEP_ID" }
+            key: { "$eval": "DEP_KEY" }
+          buckets:
+            fenix: 'net-mozaws-stage-delivery-archive'
+            focus: 'net-mozaws-stage-delivery-archive'
```

Comment:
This might need to be `focus-android` too? I really don't know though.

Reply:
yeah, let's maybe leave this til we know and we can adjust if/when we beetmove focus and find bustage.